### DISCUSSION TITLE
Can't find LDAP Authnetication logs OKD4.14

### DISCUSSION THREAD

**Discussion by ALL-SPACE-Anas** [okd-project/okd#1858] [Q&A] [answered]
Hi,

I've installed 3-master user-provisioned OKD on VMs and it's working great. I'm trying to configure LDAP authentication on it and failing to make it work. I'm following the steps defined in https://docs.okd.io/4.14/authentication/identity_providers/configuring-ldap-identity-provider.html and https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/authentication_and_authorization/configuring-identity-providers#configuring-ldap-identity-provider.

The actual problem is that I can't find any logs for LDAP authentication attempts. I've tried to enable log level debug to look at logs using the steps here https://access.redhat.com/articles/6990472 but I don't see any logs. I've also tried finding LDAP logs in pods of 'openshift-oauth-apiserver' and 'openshift-authentication-operator' but no luck. (probably I'm not the only one with the issue https://www.reddit.com/r/openshift/comments/txh1y4/how_to_debug_ldap_auth_in_okd_4/)

At the OKD login webpage, I see the option for LDAP auth so OKD must be recognizing LDAP configuration that I've added.
![image](https://github.com/okd-project/okd/assets/110912588/5422fd16-0450-48b5-98b3-b174b07fc54e)
I'm not sure what else I need to provide. Please let me know if need further information. Really appreciate any pointers in the right direction.

Thanks

**Comment by peterroth** [comment, marked as answer]
Hello,
This article may help: [User login attempts not found in the OpenShift audit log files](https://access.redhat.com/solutions/5810281).
The version is 4.6 and 4.7, but worth a try if you have a test cluster to play with.

**Reply by ALL-SPACE-Anas** [reply]
The article helped figure out what the issue was. I was testing out the configuration without waiting for the current pods to be terminated and new ones created. Couldn't find the logs since they'd be lost with the terminated pods. Thank you for your help.
